src/main_WorldCacheTest.cpp

```diff
@@ -2,64 +2,27 @@
 
 #include <BsFPSCamera.h>
 #include <Components/BsCCamera.h>
-#include <FileSystem/BsFileSystem.h>
-#include <GUI/BsCGUIWidget.h>
-#include <Resources/BsResources.h>
-#include <Scene/BsPrefab.h>
-#include <Scene/BsSceneObject.h>
 #include <Utility/BsTimer.h>
 
-#include <BsZenLib/ImportPath.hpp>
-#include <daedalus/DATFile.h>
-
 #include <core.hpp>
-#include <components/AnchoredTextLabels.hpp>
-#include <components/GameWorld.hpp>
-#include <components/Item.hpp>
-#include <components/Waynet.hpp>
-#include <components/Waypoint.hpp>
 #include <exception/Throw.hpp>
 #include <log/logging.hpp>
 #include <original-content/VirtualFileSystem.hpp>
+#include <BsZenLib/CacheUtility.hpp>
```

Comment:
Again a small include style nitpick: `BsZenLib` is a dependency, and not the code of this repo. Therefore, it should be in a separate block above this repository's code:

(At least I do it like that and think it is useful, and `clang-format` works pretty good with that scheme. What's your thoughts on stuff like that?)

```
#include <memory>

#include <BsFPSCamera.h>
#include <Components/BsCCamera.h>
#include <Utility/BsTimer.h>

#include <BsZenLib/CacheUtility.hpp>

#include <core.hpp>
#include <exception/Throw.hpp>
#include <log/logging.hpp>
#include <original-content/VirtualFileSystem.hpp>
```

Reply:
I think it's a nice way of organizing includes. I just have to get used to it!

Reply:
Great :+1: